test cases/common/132 get define/meson.build

```diff
@@ -66,6 +67,12 @@ foreach lang : ['c', 'cpp']
   have = cc.get_define('MESON_FAIL_VALUE')
   assert(have == '', 'MESON_FAIL_VALUE value is "@0@" instead of ""'.format(have))
 
+  # Check that an empty define is reported as existing.
+  have_val = cc.get_define('MESON_EMPTY_VALUE', prefix: ['#define MESON_EMPTY_VALUE'])
+  have     = cc.has_define('MESON_EMPTY_VALUE', prefix: ['#define MESON_EMPTY_VALUE'])
+  assert(have_val == '', 'MESON_EMPTY_VALUE value is "@0@" instead of ""'.format(have_val))
+  assert(have, 'MESON_EMPTY_VALUE was not found even though it should have been')
```

Comment:
This checks that the method returns `true` when the thing is defined but not that it returns `false' when it is not defined. That should be checked as well.

Reply:
Indeed, added.